packages/dev/parcel-transformer-mdx-docs/SandpackFiles.js

```diff
@@ -0,0 +1,205 @@
+const indexHtml = `<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>React Spectrum Example</title>
+  </head>
+  <body>
+      <style>
+        body {
+          min-height: 100vh;
+          margin: 0;
+        }
+        #app {
+          min-height: 100vh;
+        }
+        #app > div {
+          padding: 40px;
+          min-height: calc(100vh - 80px);
+        }
+      </style>
+    <div id="app"></div>
+  </body>
+</html>`;
+
+const sandpackImports = `import {
+  SandpackProvider,
+  SandpackLayout,
+  SandpackCodeEditor,
+  SandpackPreview,
+  SandpackThemeProvider,
+  useActiveCode,
+  useSandpack,
+  useSandpackNavigation,
+  UnstyledOpenInCodeSandboxButton
+} from "@codesandbox/sandpack-react";
+import Copy from '@spectrum-icons/workflow/Copy';
+import Refresh from '@spectrum-icons/workflow/Refresh';
+import LinkOut from '@spectrum-icons/workflow/LinkOut';
+import {ActionButton, ButtonGroup, Tooltip, TooltipTrigger} from '@adobe/react-spectrum';`;
+
+const formatExampleCode = (code, fixIndention = false) => fixIndention ? code.replace(/`/g, '\\`').replace(/\n/g, '\n    ').trim() : code.replace(/`/g, '\\`').trim();
+
+const formatImports = (imports) => imports.join('\n').replace(/`/g, '\\`');
+
+const getIndexFile = (componentName, colorScheme = 'light') => `import React from "react";
+import { createRoot } from "react-dom/client";
+import { Provider, defaultTheme } from "@adobe/react-spectrum";
+import ${componentName} from "./${componentName}";
+
+const container = document.getElementById("app");
+const root = createRoot(container);
+root.render(
+  <Provider theme={defaultTheme} colorScheme="${colorScheme}">
+    <${componentName} />
+  </Provider>
+);
+`;
+
+const getExampleFile = (exampleCode, imports) => `${formatImports(imports)}
+
+export default function Example() {
+  return (
+    ${formatExampleCode(exampleCode, true)}
+  );
+}
+`;
+
+const getNamedExampleFile = (exampleCode, imports) => `${formatImports(imports)}
```

Comment:
missing the react import from before? Noticing this in the docs:
<img width="1477" alt="image" src="https://user-images.githubusercontent.com/9661127/180315582-5601d6ba-940b-4780-ba8e-663651543cbb.png">

Reply:
shouldn't need it with React 18?

Reply:
Yeah, I removed it because of that (and not to clutter every example), but that example uses React.useState

Reply:
Just pushed a fix to add
```
import {useState} from 'react';
```
to that example